helm/cilium/templates/cilium-ingress-service.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.ingressController.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.ingressController.service.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    cilium.io/ingress: "true"
+    {{- if .Values.ingressController.service.labels }}
+    {{- toYaml .Values.ingressController.service.labels | nindent 4 }}
+    {{- end }}
+  {{- if .Values.ingressController.service.annotations }}
+  annotations:
+    {{- toYaml .Values.ingressController.service.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    nodePort: {{ .Values.ingressController.service.insecureNodePort }}
+  - name: https
+    port: 443
+    protocol: TCP
+    nodePort: {{ .Values.ingressController.service.secureNodePort }}
+  type: {{ .Values.ingressController.service.type }}
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ .Values.ingressController.service.name }}
+  namespace: {{ .Release.Namespace }}
+  {{- if .Values.ingressController.service.labels }}
+  labels:
+    {{- toYaml .Values.ingressController.service.labels | nindent 4 }}
+  {{- end }}
+  {{- if .Values.ingressController.service.annotations }}
+  annotations:
+    {{- toYaml .Values.ingressController.service.annotations | nindent 4 }}
+  {{- end }}
+subsets:
+- addresses:
+  - ip: "192.192.192.192"
```

Comment:
what this suppose to be? is that for real for every cluster?

Reply:
https://github.com/cilium/cilium/blob/b8786b36e93daf3c779216b6f23327ff19f8cc66/install/kubernetes/cilium/templates/cilium-ingress-service.yaml#L51 can't explain that value tbh.... not sure why they are creating a manual Endpoint with that IP